components/automate-ui/src/app/pipes/pagination-true-end.pipe.ts

```diff
@@ -0,0 +1,14 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'paginationTrueEnd'
+})
+export class PaginationTrueEndPipe implements PipeTransform {
+
+  transform(pageEnd: number, totalCount: number): number {
+    if (pageEnd > totalCount) {
+      pageEnd = totalCount;
+    }
+    return pageEnd;
+  }
+}
```

Comment:
the purpose of this pipe is to make sure that the correct number of the last item is show.  If the item number passed in is larger than the total number, it gets knocked down to the total item number.  Using the pipe lets us avoid continually running a function in the template.